fe/src/components/HomeCard.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <v-app>
+      <v-main>
+        <h2>Homes</h2>
+        <v-container>
+          <v-layout>
+            <div v-for="home in homes" :key="home.id" class="pl-3 pr-11">

```

Comment:
Dont use for loop, just create one card.

Reply:
I have already resolved it. Thank you for your time and insights.